<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="iconSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="usernameLabel">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>110</y>
      <width>81</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>40</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>用户名 :</string>
    </property>
   </widget>
   <widget class="QLabel" name="passwordLabel">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>250</y>
      <width>81</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>40</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>密  码 ：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="usernameInput">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>120</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordInput">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>260</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="logBtn">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>350</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>登    录</string>
    </property>
   </widget>
   <widget class="QLabel" name="welcomLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>欢</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>90</y>
      <width>111</width>
      <height>381</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="welcomLabel_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>160</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>迎</string>
    </property>
   </widget>
   <widget class="QLabel" name="welcomLabel_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>280</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>登</string>
    </property>
   </widget>
   <widget class="QLabel" name="welcomLabel_4">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>390</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>录</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
